spec/rubocop/cop/performance/push_splat_spec.rb

```diff
@@ -0,0 +1,62 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Performance::PushSplat do
+  subject(:cop) { described_class.new }
+
+  before do
+    inspect_source(cop, source)
+  end
+
+  shared_examples_for 'push splat' do |pushed|
```

Comment:
I think it would be good to have a test for `push(*)` that is not called on an object (the receiver being nil).